src/util/duration.cpp

```diff
@@ -18,7 +18,11 @@ static const qint64 kSecondsPerDay = 24 * kSecondsPerHour;
 } // namespace
 
 // static
-QString DurationBase::formatSeconds(double dSeconds, Precision precision) {
+QChar DurationBase::kCentisecondSeperator = QChar(0x2009);
+QChar DurationBase::kHectosecondSeperator = QChar(0x231E);
+
+// static
+QString DurationBase::formatTime(double dSeconds, Precision precision) {
     if (dSeconds < 0.0) {
```

Comment:
use VERIFY_OR_DEBUG_ASSERT macro

Reply:
@Be-ing when I replace these I get `Critical [Main]: DEBUG ASSERT: "dSeconds < 0.0" in function static QString mixxx::DurationBase::formatTime(double, mixxx::DurationBase::Precision) at src/util/duration.cpp:26` when I attempt to run. Any ideas?